app/views/layouts/_google_analytics_script.haml

```diff
@@ -1,73 +1,31 @@
+    = render partial: "analytics/google_tag_manager"
 
   - if !feature_enabled?(:only_gtm)
-    var _gaq = _gaq || [];
-    = "_gaq.push(['_setAccount', '#{APP_CONFIG.google_analytics_key}']);"
-    = "_gaq.push(['_setDomainName', '.#{PublicSuffix.parse(request.host).domain}']);"
-    _gaq.push(
-    ['_addIgnoredOrganic', 'sharetribe'],
-    ['_addIgnoredOrganic', 'sharetribe.com'],
-    ['_addIgnoredOrganic', 'www.sharetribe.com'],
-    ['_addIgnoredOrganic', 'kassi'],
-    ['_addIgnoredOrganic', 'share tribe'],
-    ['_addIgnoredOrganic', 'kassi.eu'],
-    ['_setAllowLinker', true],
-    ['_trackPageview'],
-    ['_trackPageLoadTime']
-    );
-
+    = render partial: "analytics/legacy_google_analytics"
 
   - if @current_community && @current_community.google_analytics_key
-    - if feature_enabled?(:customer_universal_analytics)
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga_customer');
-
-      = "var community_ga_key = '#{@current_community.google_analytics_key}';"
-      = "var community_cookie_domain = '#{PublicSuffix.parse(request.host).domain}';"
-      ga_customer('create', community_ga_key, 'auto', {'legacyCookieDomain': community_cookie_domain, 'allowLinker': true});
-      ga_customer('send', 'pageview');
-    - else
-      var _gaq = _gaq || [];
-      = "_gaq.push(['b._setAccount', '#{@current_community.google_analytics_key}']);"
-      = "_gaq.push(['b._setDomainName', '.#{PublicSuffix.parse(request.host).domain}']);"
-      = "_gaq.push(['b._addIgnoredOrganic', '#{Maybe(@current_community.name(I18n.locale)).gsub("'","").or_else("")}']);"
-      = "_gaq.push(['b._addIgnoredOrganic', '#{@current_community.domain || @current_community.ident}']);"
-      _gaq.push(
-      ['b._setAllowLinker', true],
-      ['b._trackPageview'],
-      ['b._trackPageLoadTime']
-      );
-
-    var secondary_analytics_in_use = true;
+    = render partial: "analytics/customer_analytics"
 
   - unless(feature_enabled?(:only_gtm) && feature_enabled?(:customer_universal_analytics))
-    (function() {
-    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-    })();
-
+    :javascript
+      (function() {
+      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
+      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
+      })();
```

Comment:
I guess, this could be added already in legacy_google_analytics? 
(Isn't this something that needs to be there when we or marketplace admins use legacy analytics.)